velox/docs/functions/spark/array.rst

```diff
@@ -51,6 +51,17 @@ Array Functions
         SELECT array_min(ARRAY [4.0, float('nan')]); -- 4.0
         SELECT array_min(ARRAY [NULL, float('nan')]); -- NaN
 
+.. spark:function:: array_repeat(element, count) -> array(E)
+
+    Returns an array containing ``element`` ``count`` times. If ``count`` is negative or zero,
+    returns empty array. If ``element`` is NULL, returns an array containing ``count`` NULLs.
+    Throws an exception if ``count`` exceeds the max limit, i.e., MAX_INT32 - 15.  ::
```

Comment:
Is there a limit for negative count to throw exception? Can we update the description and example for count being null?

Reply:
> Is there a limit for negative count to throw exception? 

No, all negative count will return empty array, like count = 0. 
> Can we update the description and example for count being null?

Just updated the doc and we have a test for NULL count.